gef-ui/server/utils/helpers.js

```diff
@@ -239,6 +240,50 @@ const isUkefReviewPositive = (applicationStatus) => {
   return acceptable.includes(applicationStatus);
 };
 
+const getFacilitiesAsArray = (facilities) => facilities.items.filter(({ details }) => !details.coverDateConfirmed).map(({ details }) =>
+  [
+    { text: details.name },
+    { text: details.ukefFacilityId },
+    { text: `${details.currency} ${details.value.toLocaleString('en', { minimumFractionDigits: 2 })}` },
+    { html: `<a href = '/gef/application-details/${details.applicationId}/${details._id}/ukef-cover-start-date' class = 'govuk-button govuk-button--secondary govuk-!-margin-0'>Update</a>` },
+  ]);
```

Comment:
Do you think we should rename or add a comment so it's more explicit it's mapping for the `govukTable`? 

Not sure what could rename it to 🤔

Reply:
Good shout, have added comments.